script/install_debian_dependencies.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+set -o errexit
+set -o nounset
```

Comment:
should these be set in `install_dependencies.sh` and `install_macos_dependencies.sh` as well?